wolox-books/src/components/Login.vue

```diff
@@ -1,28 +1,28 @@
 <template lang='pug'>
-  form.column.container
-    img.wolox-img(src='@/assets/LogoWolox.png' @submit.prevent='onSubmit')
+  form.column.container(@submit.prevent='submit')
+    img.wolox-img(src='@/assets/LogoWolox.png')
     span.books-title
       | BOOKS
-    label.label
+    label.label(for='email')
       | Email
-    input.input(type='text' v-model='$v.email.$model')
-    .error(v-if="$v.email.$error")
+    input.input#email(type='text' v-model='$v.email.$model')
```

Comment:
Add 'id='email'`. The value of the attribute `for` of the label must match the `id` of the input.